src/lib/ecore_audio/ecore_audio_pulse.c

```diff
@@ -14,6 +14,10 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)